ruby/lib/ThreatExchange.rb

```diff
@@ -17,27 +17,14 @@ def initialize(access_token=nil)
       @baseurl = 'https://graph.facebook.com'
     end
 
-    def malware_analyses(filter={})
-      filter[:access_token] = @access_token
-      begin
-        response = RestClient.get "#{@baseurl}/malware_analyses",
-        { params: filter  }
-        result = JSON.parse(response)
-      rescue => e
-        puts e.inspect
-      end
+    def malware_analyses(params={})
+      result = get "#{@baseurl}/malware_analyses", params
       data = result['data']
       cursor = result.fetch('paging', {}).fetch('cursor', {}).fetch('after', nil)
 
       until cursor.nil?
-        filter[:after] = cursor
-        begin
-          response = RestClient.get "#{@baseurl}/malware_analyses",
-          { params: filter }
-          result = JSON.parse(response)
-        rescue => e
-          puts e.inspect
-        end
+        params[:after] = cursor
+        result = get "#{@baseurl}/malware_analyses", params
```

Comment:
This style was there before, but for consistency would be better to use `()` in this method invocation similar to all the other method invocations in this code.